lib/gemoji.rb

```diff
@@ -1 +1,2 @@
 require 'emoji'
+require 'railtie' if defined? ::Rails::Railtie
```

Comment:
This looks alright, but I'm not sure if we need a railtie at all. How about simply calling `Emoji.all` at the end of `emoji.rb`?